stock_picking_procure_method/views/stock_picking_views.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record model="ir.ui.view" id="stock_picking_form_view">
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form" />
+        <field name="arch" type="xml">
+            <field name="scheduled_date" position="after">
```

Comment:
```suggestion
            <field name="date_deadline" position="before">
```
Scheduled date is nested inside some other tags. To avoid inserting this field inside those, hook to this one which has a regular declaration.

Reply:
> Scheduled date is nested inside some other tags. To avoid inserting this field inside those, hook to this one which has a regular declaration.

Changes made, as per your request.

Reply:
And the other change?

Reply:
> And the other change?

It was already sorted.